### ISSUE TITLE
window is not defined

### ISSUE THREAD

**Issue by huytrongnguyen** [huytrongnguyen/roxie#3]
```
Trace: 2019-03-31T08:58:37 [ERROR] window is not defined caused by:
    at Object.exports.logError (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\core\lib\logger.js:6:57)
    at Object.<anonymous> (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:91:52)
    at step (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:43:23)
    at Object.next (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:24:53)
    at D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:18:71
    at new Promise (<anonymous>)
    at __awaiter (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:14:12)
    at D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:77:83
    at Layer.handle [as handle_request] (D:\Study\huytrongnguyen\fanza-library\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Study\huytrongnguyen\fanza-library\node_modules\express\lib\router\route.js:137:13)
[nodemon] files triggering change check: src\pages\shared\navigation.tsx
[nodemon] matched rule: D:\Study\huytrongnguyen\fanza-library\src/**/*
[nodemon] changes after filters (before/after): 1/1
[nodemon] restarting due to changes...
[nodemon] src\pages\shared\navigation.tsx

[nodemon] starting `yarn restart`
[nodemon] spawning
[nodemon] child pid: 4980
$ yarn build && yarn start
$ tsc
$ node dist
GET      /
2019-03-31T09:00:07 [INFO] Content root path: D:\Study\huytrongnguyen\fanza-library\dist
2019-03-31T09:00:07 [INFO] Now listening on: http://localhost:5000
2019-03-31T09:00:07 [INFO] Application started. Press Ctrl+C to shut down.
Trace: 2019-03-31T09:00:08 [ERROR] window is not defined caused by:
    at Object.exports.logError (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\core\lib\logger.js:6:57)
    at Object.<anonymous> (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:91:52)
    at step (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:43:23)
    at Object.next (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:24:53)
    at D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:18:71
    at new Promise (<anonymous>)
    at __awaiter (D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:14:12)
    at D:\Study\huytrongnguyen\fanza-library\node_modules\@roxie\server\lib\index.js:77:83
    at Layer.handle [as handle_request] (D:\Study\huytrongnguyen\fanza-library\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Study\huytrongnguyen\fanza-library\node_modules\express\lib\router\route.js:137:13)
```

**Comment by huytrongnguyen** [comment]
Root cause of this issue from `@roxie/components`, it doesn't work on server side rendering:
```
currentRoute = window.location.hash.substring(1) || '/',
```

**Comment by huytrongnguyen** [comment]
Fixed in `@roxie/components` 0.9.5
